eng/scripts/Language-Settings.ps1

```diff
@@ -23,8 +23,8 @@ function Get-GoModuleVersionInfo($modPath)
     }
   }
 
-  LogWarning "Unable to find version for $modPath"
-  return $null
+  LogWarning "Unable to find version for $modPath, considered to be onboard rp package"
```

Comment:
This is very specific to the mgmt libraries. Can we not just fix the missing versioning instead trying to fake the version?